### DISCUSSION TITLE
Please link directly to /categories/lists

### DISCUSSION THREAD

**Discussion by adlai** [community/community#57653] [Other Feature Feedback, Questions, & Ideas] [answered]
After clicking Stars in the profile, and link to share feedback, the browser tab lands in the main discussion; adding the two words "star list" to the search, without quotes, helped me find the specific category where feedback belongs.

Please link [directly to the category](https://github.com/orgs/community/discussions/categories/lists)!

**Comment by Ozgurka** [comment]
My name ozzy ![cmcshare1691606282798.jpg](https://github.com/community/community/assets/116981212/c6024720-0a38-4781-844f-a723f92d706b)

![Mon Jul 31 22_22_22 GMT+03_00 2023.jpg](https://github.com/community/community/assets/116981212/3ee9927a-aea0-42aa-9a92-d15c5fd13117)

![Uploading IMG-20230810-WA0002.jpeg …]()

![unnamed.png](https://github.com/community/community/assets/116981212/bbf95c5a-2}01b-4a01-91b4-e4509b288298)

![cmcshare1691606340939.jpg](https://github.com/community/community/assets/116981212/7cbb11e7-931f-4bb9-bce1-d7a01aa26e4a)

![Uploading IMG-20230810-WA0004.jpeg …]()

![QR Scanner_Antalya _1691431173860.jpg](https://github.com/community/community/assets/116981212/8b1e93d4-264f-4aff-90a4-6dc9313bc24a)

![cmcshare1691606314401.jpg](https://github.com/community/community/assets/116981212/cf2415d1-8eb7-4c85-a2b7-ede818b60ad1)

**Comment by Ozgurka** [comment]
Sergio kim bilmiyorum

**Comment by Ozgurka** [comment]
Umarım tamam

**Reply by Ozgurka** [reply]
Ne yapmalıyım

**Reply by NasrErekat** [reply]
> Umarım tamam

**Comment by adlai** [comment]
Seeing that this has not been closed yet, I checked the current state: the link does appear to go into one of the discussions, although then the browser tab gets redirected to the main page again.

I have not conducted any sort of detailed investigation; after all, I don't work for GitHub...

**Comment by adlai** [comment, marked as answer]
> After [clicking Stars in the profile](https://github.com/adlai?tab=stars), and link to share feedback, [...]

It appears that the feedback solicitation has vanished, thus the discussion should be closed.
